src/state/migrations/1.ts

```diff
@@ -0,0 +1,24 @@
+import { DEFAULT_DEADLINE_FROM_NOW } from 'constants/misc'
+import { PersistState } from 'redux-persist'
+import { UserState } from 'state/user/reducer'
+
+export type PersistAppStateV1 = {
+  _persist: PersistState
+} & { user?: UserState }
+
+/**
+ * Migration to change the default user deadline from 30 minutes to 10 minutes.
+ * We only migrate if the saved deadline is the old default.
+ */
+export const migration1 = (state: PersistAppStateV1 | undefined) => {
+  if (state?.user && state.user?.userDeadline === 1800) {
```

Comment:
qq: we don't track if a user has set their deadline to be 1800 manually right?

Reply:
no, we just persist the deadline all the time and 1800 is the initial/default value.